node/tests/GlideClient.test.ts

```diff
@@ -1460,70 +1459,6 @@ describe("GlideClient", () => {
         TIMEOUT,
     );
 
-    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
```

Comment:
Do we think that this test is not relevant at all?
Is it true for what @shohamazon is working on as well?